tests/utils/retry.ts

```diff
@@ -0,0 +1,20 @@
+export const retry = async (callback: () => Promise<any>, timeout = 3000) => {
```

Comment:
Maybe we need to add some sleep between retries.

Reply:
done